src/types.ts

```diff
@@ -0,0 +1,59 @@
+export type Experiments = {
+	allocations: Allocation[];
+	exclusions: string[];
+	confirmations: Confirmation[];
+}
+
+export interface Allocation {
+	cid: string;
+	eid: string;
+	excluded: boolean;
+	group_id: string;
+	ordinal: number;
+	uid: string;
+}
+
+export interface Confirmation {
+	cid: string;
+	timestamp: number;
+}
+
+export interface Evolv {
+	client: {
+		getDisplayName: (type: string, id: string) => Promise<string>;
+		on: (event: string, callback: () => void) => void;
+	};
+	context: {
+		get: (key: string) => Experiments;
+		remoteContext: RemoteContext;
+	}
+}
```

Comment:
I don't know much we should care, but we could pull these from one of our packages.